src/net/socket.rs

```diff
@@ -177,12 +213,12 @@ impl Socket {
 
     /// Set the link conditioner for this socket. See [LinkConditioner] for further details.
     pub fn set_link_conditioner(&mut self, link_conditioner: Option<LinkConditioner>) {
-        self.link_conditioner = link_conditioner;
+        self.socket_wrapper.set_link_conditioner(link_conditioner);
     }
 
     /// Get the local socket address
```

Comment:
```suggestion
    /// Returns the local socket address
```

Reply:
It is not your commit, but I prefer 'returns' so I thought I might as well suggest this.